src/tickets.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Tickets
+---
+
+<a href="https://pretix.eu/pyconuk/PyConUK2022" target="_blank">Buy your PyCon UK 2022 tickets now</a>
+
+<div class="box box_yellow">
+  <h3>Please note</h3>
+  <p>As part of our return we are temporarily only offering tickets that cover the full conference. We are hoping to return to usual in 2023!</p>
```

Comment:
let's get rid of 'temporarily' so people don't think we might change later this year.

Reply:
Removed.